test/functional/wallet_conflicts.py

```
@@ -0,0 +1,127 @@
+Test that wallet correctly tracks transactions that have been conflicted by blocks, particularly during reorgs.
+"""
+
+from decimal import Decimal
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+        assert_equal,
+)
+
+class TxConflicts(BitcoinTestFramework):
+    def add_options(self, parser):
```

Ruff (0.12.2)

Missing return type annotation for public function `add_options`

Add return type annotation: `None`

(ANN201)